less/hotgrid.less

```diff
@@ -22,6 +22,7 @@
 	.hotgrid-item-image {
 		display: block;
 		margin: 10px;
+		width: 100%;
```

Comment:
I'd recommend changing this so that margin is 2.5% and width is 95% as the above overflows the container div.